src/main/java/edu/kis/powp/jobs2d/drivers/adapter/AbstractLineDrawerAdapter.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class AbstractLineDrawerAdapter extends AbstractDriver
+{
+  private final Job2dDriver job2dDriver;
+
+  public AbstractLineDrawerAdapter(Job2dDriver job2dDriver)
+  {
+    super(0, 0);
+    this.job2dDriver = job2dDriver;
+  }
+
+  public static AbstractDriver toAbstractDriver(Job2dDriver job2dDriver)
+  {
+    return new AbstractLineDrawerAdapter(job2dDriver);
+  }
+  // todo: no way to reimplement setPosition for Job2d drivers (especially for LoggerDriver)
+
+  @Override
+  public void operateTo(int x, int y)
+  {
+    job2dDriver.operateTo(x, y);
+  }
```

Comment:
AbstractDriver already has some implementation, which should be used.
Direct SetPosition calls are ignored so this Adapter would not work well.